.vscode/launch.json

```diff
@@ -1,19 +1,25 @@
 {
-	"version": "0.2.0",
-	"configurations": [
-		{
-			"name": "ra2",
-			"type": "clr",
-			"linux": {
-				"type": "mono"
-			},
-			"osx": {
-				"type": "mono"
-			},
-			"request": "launch",
-			"program": "${workspaceRoot}/engine/OpenRA.Game.exe",
-			"cwd": "${workspaceRoot}",
-			"args": ["Engine.LaunchPath=${workspaceRoot}", "Engine.ModSearchPaths=${workspaceRoot}/mods,./mods", "Game.Mod=ra2", "Debug.DisplayDeveloperSettings=true"]
-		},
-	]
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Red Alert 2",
+            "type": "clr",
+            "linux": {
+                "type": "mono",
+            },
+            "osx": {
+                "type": "mono",
+            },
+            "request": "launch",
+            "preLaunchTask": "build",
+            "program": "${workspaceRoot}/engine/OpenRA.Game.exe",
+            "cwd": "${workspaceRoot}/engine/",
+            "args": [
+                "Engine.LaunchPath=${workspaceRoot}",
+                "Engine.ModSearchPaths=${workspaceRoot}/mods",
```

Comment:
This is dropping a value.

Reply:
I meant to comment on this when making the PR but forgot.
In my testing the now-dropped value wasn't needed.
Do we know why it was there previously?

Reply:
It is the mods directory of the engine directory. We probably need it for the modcontent mod.

Reply:
> We probably need it for the modcontent mod.
Ah, you are correct.  I didn't think to test that.
I've added it back in (and switched indent to tabs).